.github/workflows/windows.yml

```diff
@@ -400,24 +402,31 @@ jobs:
       - name: Download OpenVINO package
         uses: actions/download-artifact@v4
         with:
-          name: openvino_package
+          name: ${{ env.OV_ARTIFACT_NAME }}
           path: ${{ env.INSTALL_DIR }}
 
 
       - name: Extract OpenVINO packages
+        if: needs.openvino_build.result == 'success'
         run: |
           pushd ${{ env.INSTALL_DIR }}
             Expand-Archive openvino_package.zip -DestinationPath ./tmp
             mv ./tmp/*/* .
           popd
 
-      - name: Install OpenVINO Python wheel
+      - name: Install OpenVINO Python wheel from build
         if: needs.openvino_build.result == 'success'
         run: |
           # Find and install wheel
           $ovCoreWheelPath=Get-ChildItem -Path "${{ env.INSTALL_DIR }}\\tools" -Filter openvino-*.whl | % { $_.FullName }
           python3 -m pip install "$ovCoreWheelPath"
 
+      - name: Install OpenVINO Python wheel from pre-built artifacts
+        if: needs.openvino_download.outputs.status == 'success'
+        run: |
+          python3 -m pip install ${{ needs.openvino_download.outputs.ov_wheel_input }}
```

Comment:
Decided to remove ov_wheel_input argument and use only ov_wheel_source (--extra-url <channel>, --find_links <path>)